src/main/drivers/light_ws2811strip.h

```diff
@@ -24,30 +24,12 @@
 // for 50us delay
 #define WS2811_DELAY_BUFFER_LENGTH 42
 
-#define WS2811_DATA_BUFFER_SIZE (WS2811_BITS_PER_LED * WS2811_LED_STRIP_LENGTH)
+#define WS2811_DATA_BUFFER_SIZE    (WS2811_BITS_PER_LED * WS2811_LED_STRIP_LENGTH)
 // number of bytes needed is #LEDs * 24 bytes + 42 trailing bytes)
-#define WS2811_DMA_BUFFER_SIZE  (WS2811_DATA_BUFFER_SIZE + WS2811_DELAY_BUFFER_LENGTH)
+#define WS2811_DMA_BUFFER_SIZE     (WS2811_DATA_BUFFER_SIZE + WS2811_DELAY_BUFFER_LENGTH)
 
-#if defined(STM32F40_41xxx)
-#define WS2811_TIMER_HZ        84000000
```

Comment:
Some targets are using one timer for (PPM input or motor output) and LED_STRIP. This will break PPM/MOTOR.

@blckmn, BLUEJAYF4 uses TIM3 for PPM and LED_STRIP. LOL. Please check with PPM enabled.